hyperdash/experiment.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .client import HDClient
+from .monitor import monitor
+from .io_buffer import IOBuffer
+from .server_manager import ServerManagerHTTP
+from .hyper_dash import HyperDash
+from .utils import get_logger
+
+import sys
+import uuid
+import threading
+from six.moves.queue import Queue
+# Python 2/3 compatibility
+__metaclass__ = type
+
+class ExperimentRunner:
```

Comment:
A docstring here explaining how this class is basically a No-op so we can reuse the Hyperdash class would be nice